python/src/aiconfig/batch_execution/mvp.py

```diff
@@ -0,0 +1,85 @@
+from typing import Dict
```

Comment:
nit: I learned recently learned this is deprecated (https://docs.python.org/3/library/typing.html#deprecated-aliases)